python/paddle/fluid/tests/unittests/test_cross_entropy_loss.py

```diff
@@ -0,0 +1,136 @@
+# Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
+
+import paddle
+import paddle.fluid as fluid
+import numpy as np
+import unittest
+
+
+class CrossEntropyLoss(unittest.TestCase):
+    def test_cross_entropy_loss_mean(self):
+        input_np = np.random.random([3, 5]).astype(np.float32)
```

Comment:
See:

https://github.com/PaddlePaddle/Paddle/wiki/OP-test-input-shape-requirements

the shape in unit testing is too small

Reply:
DONE